resources/entity_tree_form.ui

```diff
@@ -42,9 +42,6 @@
      </property>
      <item>
       <widget class="QComboBox" name="task_status_combo">
-       <property name="text">
-        <string>Task Status</string>
-       </property>
```

Comment:
Just checking that we meant to remove this property?

Reply:
This PR intends to leave the generated python UI files as #133. 

Since you were the author of that PR, let us know if we can remove it finally or not.

Reply:
Hm forgive me, I don't remember removing this or not, since that PR was from last year. Looking at the PR though, there were no modifications to this file `resources/entity_tree_form.ui`

Reply:
I did some historical GIT research and found out that this block was introduced on #147 manually updating the generated files.

I'll do some tests and put back this block of code if needed. Thanks for the heads up

Reply:
Update: this block generates [this line](https://github.com/shotgunsoftware/tk-multi-workfiles2/pull/150/commits/f5d88b2f82b9749e3dbd5efab3db0195aab2184f#diff-d440001cd0e979906cf2f3bf25d9c39c186ef81cb6ecb5a677d113c2ed6cf1b5L104) on `python/tk_multi_workfiles/ui/entity_tree_form.py` which breaks the layout like this

![image](https://github.com/user-attachments/assets/2bb37099-d2e3-4faa-9519-ea8c3c26f982)

The best option is to keep it out.